website/src/components/markdown/Code.js

```diff
@@ -1,46 +1,41 @@
 /** @jsx jsx */
 
-import React from 'react'; // eslint-disable-line
-import Highlight, { defaultProps } from 'prism-react-renderer';
-import reactAddonsTextContent from 'react-addons-text-content';
 import { jsx } from '@emotion/core';
 
 import theme from '../../prism-themes/custom';
 
-/*
-This solution should be seen as extremely temporary and not a platonic ideal of code highlighting.
+function themeToDict(theme, language) {
```

Comment:
This is a slight modification of the [`themeToDict()` utilitly from `prism-react-renderer`](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/utils/themeToDict.js)